src/types.jl

```diff
@@ -734,10 +750,8 @@ construction where `n` is the number of threads to be used per replica.
     value is passed to the argument `replica_general_inters`.
 - `replica_general_inters=[() for _ in 1:n_replicas]`: the general interactions for 
     each replica.
-- `constraints::CN=()`: the constraints for bonds and angles in the system (to be used if the same 
-    for all replicas). Typically a `Tuple`.
-- `replica_constraints=[() for _ in 1:n_replicas]`: the constraints for bonds and angles in each
```

Comment:
I think you can allow `replica_constraints` by calculating `df` for each system. There may be situations where people want different constraints for each replica.